usecase/tenantd.go

```diff
@@ -156,9 +157,80 @@ func (t *clientd) Start(ctx context.Context) chan []error {
 			}
 		}()
 	}
+
+	reportTicker := time.NewTicker(time.Minute)
+	go func() {
+		defer reportTicker.Stop()
+		for {
+			select {
+			case <-reportTicker.C:
+				report(t)
+			case <-ctx.Done():
+				return
+			}
+		}
+	}()
+
 	return t.server.ListenAndServe(ctx)
 }
 
+// start token cache report daemon (no need for graceful shutdown)
```

Comment:
move line 160
```suggestion
// start token cache report daemon (no need for graceful shutdown)
```

Reply:
@WindzCUHK fixed in https://github.com/AthenZ/athenz-client-sidecar/pull/18/commits/fdb26c4b3b40c9a7433675ebb0d4b68ceb23aeba